docs/faq/windows/index.md

```diff
@@ -35,12 +27,24 @@ See also [top-level FAQ page](/docs/faq/).
 wxWidgets can be used to develop and deliver applications on Windows
 XP, Vista, 7, 8, 10.
 
-<a name="winxp"></a>
+<a name="manifest"></a>
 
-### What do I need to do for Windows XP?
+### Why do controls in my application look odd?
 
-The XP manifest is included in wx/msw/wx.rc and so your application will be
-themed automatically so long as you include wx.rc in your own .rc file.
+If the controls do not look as expected (very noticeable e.g. with buttons),
+your application probably does not have an [application manifest](https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests)
+and so Windows displays the controls unthemed, as if in Windows 9x.
```

Comment:
```suggestion
and so Windows displays the controls unthemed, as if in Windows 95/98/ME.
```
Change to make this clear even to people who are not familiar with abbreviation "9x".